source/dubregistry/internal/workqueue.d

```diff
@@ -49,13 +50,35 @@ final class PackageWorkQueue {
 		}
 	}
 
+	void putFront(string pack_name)
```

Comment:
Why not implement a proper `putFront` in `FixedRingBuffer` instead?

Reply:
Because it takes ages until it's available here ;).

Reply:
Good point ;) I'll probably tag a hotfix release pretty shortly, though, to fix some issues that surfaced in conjunction with the vibe-core switch. There is just debugging of one source for network related slowdowns left.